expirationd.lua

```diff
@@ -400,6 +514,63 @@ end
 --
 -- @section Functions
 
+--- Configure expirationd.
+--
+-- How to set up a configuration option:
+--
+-- ```
+-- expirationd.cfg({metrics = true})
```

Comment:
Added:
```patch
--- a/expirationd.lua
+++ b/expirationd.lua
@@ -516,6 +516,8 @@ end
 
 --- Configure expirationd.
 --
+-- Since version 1.2.0.
+--
```